aws/session/shared_config.go

```diff
@@ -55,6 +58,9 @@ type sharedConfig struct {
 	AssumeRole       assumeRoleConfig
 	AssumeRoleSource *sharedConfig
 
+	// An external process to request credentials
+	CredentialProcess string
```

Comment:
The `sharedConfig` utility is the utility that should be solely responsible for loading the shared config file.